third_party/proto/cosmos/bank/v1beta1/query.proto

```diff
@@ -13,7 +13,7 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/bank/types";
 service Query {
   // Balance queries the balance of a single coin for a single account.
   rpc Balance(QueryBalanceRequest) returns (QueryBalanceResponse) {
-    option (google.api.http).get = "/cosmos/bank/v1beta1/balances/{address}/{denom}";
+    option (google.api.http).get = "/cosmos/bank/v1beta1/balances/{address}/by_denom";
```

Comment:
@iramiller @arnabmitra @dwedul-figure I noticed this change in the URL from `/{denom}` to `/by_denom` on our cosmos-sdk fork version [v0.45.0-pio-1](https://github.com/provenance-io/cosmos-sdk/blob/f672cad373eccafefbe58655aae8dcf90d8b1b1b/proto/cosmos/bank/v1beta1/query.proto#L16) which we are referencing in `go.mod`. Is this change intentional?

Reply:
That's in the upstream as well... they had to do this because forward slash is allowed in a denom and the `ibc/` denoms broke this.

Reply:
Ah.